public/layouts/header.php

```diff
@@ -9,7 +9,8 @@
           rel="stylesheet"
           integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB"
           crossorigin="anonymous">
-<script defer src="<?php echo url('dist/scss.js') ?>"></script>
+    <link rel="stylesheet" href="<?php echo url('dist/css/main.css') ?>">
+    <script src="https://kit.fontawesome.com/fe6cb3607a.js" crossorigin="anonymous"></script>
```

Comment:
Font awesome styles are already imported in [`main.scss`](https://github.com/mohamed-abdul-fattah/life-system/blob/fcfdcde39f3c9081fc4308ca208362aa0b140f2b/webpack/src/scss/main.scss#L4). Why do you need this font awesome script? What does it do while font awesome is a font style?.

Reply:
Same behavior, but I did it through Font awesome website, not bootstrap website